src/resolve/resolved-dns-transaction.c

```diff
@@ -158,6 +158,8 @@ DnsTransaction* dns_transaction_free(DnsTransaction *t) {
         dns_resource_key_unref(t->key);
         dns_packet_unref(t->bypass);
 
+        mfree(t->answer_ede_msg);
```

Comment:
use regular `free()` for cases like this.

(`mfree()` only exists to return NULL cast to the type of the input as return value, so that you can write `foo = mfree(foo)`. But here you don't care about that, hence just use plain free)